jwst/jump/twopoint_difference.py

```diff
@@ -19,15 +19,23 @@
 
 HUGE_NUM = np.finfo(np.float32).max
 
-def find_crs(data, gdq, read_noise, rej_threshold, nframes):
+
+#def find_crs(indata, ingdq, read_noise, rej_threshold, nframes):
+def find_crs(indata):
 
     """
     Find CRs/Jumps in each integration within the input data array.
     The input data array is assumed to be in units of electrons, i.e. already
     multiplied by the gain. We also assume that the read noise is in units of
     electrons.
     """
-
+    if isinstance(indata, tuple):
```

Comment:
What's the reason for treating the inputs as a single argument that's a tuple? Is that for the purpose of passing slices when using multiple cores?

Reply:
Fixed.  use `pool.starmap` instead of `pool.map` to get argument unpacking in the multiprocessing pool.